cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -182,24 +182,24 @@ func (command *Evaluate) Initialize(args []string) (evaluationContext *evaluate.
 
 		if command.Runtime == "docker" {
 			if _, err := exec.LookPath("docker"); err != nil {
-				command.logger.Panic("docker runtime could not be found")
+				command.logger.Panicf("docker runtime could not be found")
```

Comment:
Not sure what the what the consequences of these changes are. Why is this easier for parsing?

Reply:
The `Panicf` does nothing for parsing, just adjusting the signature. As for all the others:

`log.Info("something", "contextA", contextA, "contextB" contextB, ...)`

ends up in the JSON logs as

```json
{
    "msg": "something",
    "contextA": contextA...,
    "contextB": contextB...,
    ...
}
```

which... is nice if you parse the JSON.